tsconfig.json

```diff
@@ -3,7 +3,12 @@
 	"compilerOptions": {
 		"target": "esnext",
 		"resolveJsonModule": true,
-		"noEmit": true
+		"noEmit": true,
+		"esModuleInterop": true,
+		"baseUrl": "./",
+		"paths": {
```

Comment:
See https://github.com/bfred-it/delegate-it/pull/5#discussion_r273319386 for context.